cfpb/consumerfinance.gov
# Add Spanish translations to social sharing links

This commit adds Spanish translations for a few pieces of text related to social sharing, using Django's translation framework. These translations have been approved internally.

In the social media module:

- The default email title (rarely if ever used)
  English: Look what I found on the CFPB's site!
  Spanish: ¡Encuentre más información en la página del CFPB!

- The default email text preface, before the link
  English: Check out this page from the CFPB
  Spanish: Revise esta página del CFPB

- Headings (previously in code using an if statement and some CSS)
  English: Share & print, Share this
  Spanish: Compartir e imprimir, Compartir

- Aria labels for sharing icons
  English: Share on, Visit us on
  Spanish: Comparta esta información en, Visítenos en

At the bottom of blog pages:

- Closing text
  English: Join the conversation. Follow CFPB on Twitter and Facebook.
  Spanish: Únase a la conversación. Síganos en Twitter y en Facebook.

## How to test this PR

Using a local production dump, visit these pages to check aria-labels and default text:

- English blog page: http://localhost:8000/about-us/blog/empowering-military-consumers-all-year-long/
- Spanish blog page: http://localhost:8000/about-us/blog/opciones-recibir-beneficios-desempleo/
- Spanish sidebar page: http://localhost:8000/es/herramientas-del-consumidor/

## Screenshots

![image](https://user-images.githubusercontent.com/654645/91196477-324d5d00-e6c8-11ea-8826-641b5fd1b8aa.png)

## Notes and todos

This change doesn't touch the existing manual translation entries in the Spanish django.po file (see #5956).

## Checklist
